Adyen/Utilities/PublicKeyProvider/PublicKeyProvider.swift

```diff
@@ -1,53 +1,48 @@
 //
-// Copyright (c) 2021 Adyen N.V.
+// Copyright (c) 2022 Adyen N.V.
 //
 // This file is open source and available under the MIT license. See the LICENSE file for more info.
 //
 
-import Adyen
 import AdyenNetworking
 import Foundation
 
-/// :nodoc:
-internal protocol AnyCardPublicKeyProvider: AnyObject {
+/// Interface to fetch the client public key.
+public protocol AnyPublicKeyProvider: AnyObject {
     
     /// :nodoc:
     typealias CompletionHandler = (Result<String, Error>) -> Void
     
-    /// :nodoc:
+    /// Fetches the client public key with a closure for success and failure.
     func fetch(completion: @escaping CompletionHandler)
 }
 
-/// :nodoc:
-internal final class CardPublicKeyProvider: AnyCardPublicKeyProvider {
+/// `PublicKeyProvider` is used to fetch the client public key that is needed for encrypting data.
+public final class PublicKeyProvider: AnyPublicKeyProvider {
```

Comment:
maybe add /// :nodoc: merchants are not supposed to use `PublicKeyProvider` or care about it :-)